modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/upgrade-catch-all-check/LPD_50175.testjava

```diff
@@ -0,0 +1,32 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Micaelle Silva
+ */
+public class LPD_50175 {
+
+	public void method(long companyId, String externalReferenceCode) {
+		AccountEntryLocalServiceUtil.fetchByExternalReferenceCode(123, externalReferenceCode);
+		AccountEntryLocalServiceUtil.fetchAccountEntryByExternalReferenceCode(externalReferenceCode, companyId);
+		AccountEntryServiceUtil.fetchByExternalReferenceCode(123, externalReferenceCode);
+		AccountEntryServiceUtil.fetchAccountEntryByExternalReferenceCode(externalReferenceCode, companyId);
+		_accountEntryLocalService.fetchByExternalReferenceCode(123, companyId);
+		_accountEntryLocalService.fetchAccountEntryByExternalReferenceCode(externalReferenceCode, companyId);
+		_accountEntryService.fetchByExternalReferenceCode(123, companyId);
+		_accountEntryService.fetchAccountEntryByExternalReferenceCode(externalReferenceCode, companyId);
```

Comment:
@MicaelleSilvaa isn't necessary only one test of failure for json Object, so just the first line its fine, remove the others.